.gitignore

```diff
@@ -1,4 +1,3 @@
 .idea/
 *.test
 .envrc
-pctl
```

Comment:
Haha, yes, I've gotten bitten by this. That's what you get for having a folder named the same thing as the binary. :)))

On that note, how about using goreleaser to create multiple binaries under build?

Reply:
Its so annoying!!!!!

that sounds good! I think we need to setup a release process for `pctl` anyway, having it as part of that work would be great. We do the same thing in eksctl